index.html

```diff
@@ -68,56 +68,56 @@ <h2>Biography</h2>
             </p>
           </div>
         </div>
-        <div class="table-body">
-          <table>
-            <caption>
+        <div class="table__table-body">
+          <table class="table">
+            <caption class="table__caption">
               CV
             </caption>
             <tbody>
               <tr>
-                <th>Year</th>
-                <th>Station</th>
+                <th class="table__th">Year</th>
+                <th class="table__th">Station</th>
               </tr>
               <tr>
-                <td class="left-column">01/2022 - now</td>
-                <td class="right-column">Bootcamp Web Development</td>
+                <td class="table__left-column">01/2022 - now</td>
+                <td class="table__right-column">Bootcamp Web Development</td>
               </tr>
               <tr>
-                <td class="left-column">04/2017 - 12/2021</td>
-                <td class="right-column">work experience #2</td>
+                <td class="table__left-column">04/2017 - 12/2021</td>
+                <td class="table__right-column">work experience #2</td>
               </tr>
               <tr>
-                <td class="left-column">08/2014 - 03/2017</td>
-                <td class="right-column">work experience #1</td>
+                <td class="table__left-column">08/2014 - 03/2017</td>
+                <td class="table__right-column">work experience #1</td>
               </tr>
               <tr>
-                <td class="left-column">09/2009 - 07/2014</td>
-                <td class="right-column">Bachelor of Science</td>
+                <td class="table__left-column">09/2009 - 07/2014</td>
+                <td class="table__right-column">Bachelor of Science</td>
               </tr>
               <tr>
-                <td class="left-column">06/2009</td>
-                <td class="right-column">Abitur</td>
+                <td class="table__left-column">06/2009</td>
+                <td class="table__right-column">Abitur</td>
```

Comment:
what u think about the class `table__td` for all the `td` tags and a second class of` table__td--right` and `table__td--left`
for `<td class="table__td table__td--left>06/2009</td>`
so u could first style all `td` tags and if you want to style a left or right td  tag by it self